src/UI/PageState/index.tsx

```diff
@@ -26,20 +28,23 @@ export const PageState = ({
   description,
   iconBgClass,
   iconSize = '5x',
-  className = 'page-state',
+  className = globalStyles.pageState,
   shouldRenderDefaultCss = true
 }: PageStateProps) => {
   const generatedClasses = getGeneratedClasses(
     className,
     shouldRenderDefaultCss,
     {
-      wrapper: 'state m-auto p-4 text-center',
-      iconContainer: classNames('icon-state mx-auto', {
-        [`${iconBgClass}`]: Boolean(iconBgClass)
-      }),
+      wrapper: `${styles.state} ${globalStyles.mAuto} ${globalStyles.p4} ${globalStyles.textCenter}`,
```

Comment:
I'm wondering if using classNames package in here won't improve the code style?
btw, you can enumerate all the styles inside an array and the classNames package will transform it accordingly